README.md

```diff
@@ -20,15 +19,11 @@ For information, see "Module names and descriptions" at
 https://terraform-ibm-modules.github.io/documentation/#/implementation-guidelines?id=module-names-and-descriptions
 -->
 
-TODO: Replace this with a description of the modules in this repo.
```

Comment:
Can you add a description of the module here just like it says in the template? Maybe point to the watsonx.data official docs too?